site/_quarto.yml

```
@@ -76,18 +76,19 @@ website:
 
     - title: "About"
       contents:
-        - about/overview.qmd
+        - text: "About ValidMind"
```

```suggestion
        - text: "About"
```

(To match what the navbar says.)

---

Also rename the "What is ValidMind?" title in the topic itself to "About ValidMind".